Managers/UInputMan.cpp

```diff
@@ -837,8 +839,19 @@ namespace RTE {
 			return;
 		}
 
-		if (g_InActivity) {
-			if (AnyStartPress(false)) {
+		if (g_InActivity) {	
+			bool BuyMenuIsOpen = 0;
+			GameActivity* temp = dynamic_cast<GameActivity*>(g_ActivityMan.GetActivity());
```

Comment:
Please never name anything temp or something like that, even with simple stuff like this. Variable names should explain what they are, and temp explains nothing except that it's temporary. I'd call this something like  `currentGameActivity`.